media-libs/libdmtx/libdmtx-0.7.4.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="Library to read and write Data Matrix barcodes"
+DESCRIPTION="libdmtx is a software library that enables programs to read and \
+write Data Matrix (or DataMatrix) 2D barcodes. It runs on several platforms, \
+and can be accessed by multiple languages."
+HOMEPAGE="http://libmtx.sourceforge.net/"
+COPYRIGHT="2005-2011 Mike Laughton"
```

Comment:
The file LICENSE in the source tarball says 2005-2011 so I used that. I even included it as the license file, along with the recipe.